packages/core/src/amazonq/lsp/util.ts

```diff
@@ -0,0 +1,10 @@
+/*!
```

Comment:
These various fetch/download/lsp utils seem potentially applicable to both Toolkit and Q in the future. How much of these util modules are Q-specific? Can most of them live in /core/ ?

Reply:
All the logic for downloading/fetching etc is in core. The only thing that should be in the q lsp folder is the codewhisperer language server resolver and the codewhisperer language server activation

Reply:
oh right this is in core but the amazonq part of core... if it's not q-specific then it could live in shared.